docs/HowToRunSTDKSample.md

```diff
@@ -0,0 +1,254 @@
+
+Open a terminal window and run the following stdk-keygen command to create a Device Identity.   
+
+```bash
+$ cd external/stdk/st-device-sdk-c/tools/keygen/linux
+$ ./stdk-keygen -h
+usage: stdk-keygen -m MNID [-f firmware_version]
+
+$ ./stdk-keygen -m **** -f V201910       # replace '****' with your MNID
+Go Device Identity of Developer Workspace.
+
+Serial Number:
+┌──────────┬──────────┐
+│ STDK**** │ ce**2**3 │
+└──────────┴──────────┘
+
+Public Key:
+1D********a21F********8WwP********yU/n8vFvM=
+
+$ tree
+.
+├── ed25519.pubkey
+├── ed25519.seckey
+└── output_STDK****ce**2**3
+    └── device_info.json
+```
+Copy the Serial Number and Public Key after running the command on your computer. You will need to upload those values to the SmartThings Cloud via [Developer Workspace](https://smartthings.developer.samsung.com/).
+
+After you create a device identity with a command with an option like above, You can get the ready-to-use device_info.json file directly.   
+
+## Register Device Profile on SmartThings Devworkspace   
+### Sing in to [Devworkspace](https://smartthings.developer.samsung.com/).   
+Select 'Sing In' where right-top on the page.   
+![Select Sign In](./media/STDK_Selection_Sign_in.png)   
+![Select Sign In](./media/STDK_Singin_devworkspace.png)   
+### Create a Project on SmartThings Cloud.   
+You need to register a new project to SmartThings Cloud.   
+
+1. Create a Project.   
+    1. Select 'New Project' to regist new project(or device).   
+![Select New Project](./media/STDK_Create_New_Project.png)   
+    1. Select 'Device Integration'.   
+![Select 'Device Integration'](./media/STDK_Selection_Device_Integration.png)   
+    1. Select 'Direct-Connected' type.   
+![Select Direct-Connected](./media/STDK_Selection_Direct_Connected.png)   
+    1. Input Device name and Select 'CREATE PROJECT'.   
+![Input Device name and Select 'CREATE PROJECT'.](./media/STDK_Create_Project_Direct_Connected.png)   
+1. Create a Device Profile.   
+    A device profile contains the components, capabilities, and metadata (ID, name, etc.). This information defines the actions and attributes that an IoT device can perform. Click the "GO TO DEVICE PROFILE" and then, enter the all remaining information required.   
+
+    1. Select 'Add a Device Profile'.   
+![Select the 'GO TO DEVICE PROFILE'](./media/STDK_Add_A_Device_Profile_1.png)  
+![Select the 'Add a Device Profile'](./media/STDK_Add_A_Device_Profile_2.png)
+    1. Select 'Add a Device Profile'.   
+    A device profile contains the components, capabilities, and metadata (ID, name, etc.). This information defines the actions and attributes that an IoT device can perform.   
```

Comment:
1. There seems repetition in points.
2. You already mentioned what a device profile contains, so you can remove it here.

Reply:
Has Changed.